catroid/src/main/java/org/catrobat/catroid/content/bricks/SpeakAndWaitBrick.java

```diff
@@ -23,48 +23,25 @@
 
 package org.catrobat.catroid.content.bricks;
 
-import android.content.Context;
-import android.view.View;
-import android.widget.TextView;
-
 import org.catrobat.catroid.R;
 import org.catrobat.catroid.content.Sprite;
 import org.catrobat.catroid.content.actions.ScriptSequenceAction;
 import org.catrobat.catroid.content.actions.SpeakAction;
 import org.catrobat.catroid.formulaeditor.Formula;
-import org.catrobat.catroid.ui.fragment.FormulaEditorFragment;
 
-import java.io.File;
 import java.util.List;
 
 public class SpeakAndWaitBrick extends FormulaBrick {
 
 	private static final long serialVersionUID = 1L;
-	private transient View prototypeView;
-
-	File speechFile;
-	private float duration;
```

Comment:
Don't forget to omit these fields in the XstreamSerializer.

Reply:
sure :wink I will also increase the language version.

Reply:
done.